src/android/shakeNotifier.java

```diff
@@ -0,0 +1,246 @@
+package edu.berkeley.eecs.emission.cordova.shakeDetect;
```

Comment:
Does this even work? I thought java required filenames to match classnames, so this would be `ShakeNotifier`.